### DISCUSSION TITLE
训练ch_PP-OCRv4几步之后卡住

### DISCUSSION THREAD

**Discussion by jindameias** [PaddlePaddle/PaddleOCR#13777] [Q&A]
### 🔎 Search before asking

- [X] I have searched the PaddleOCR [Docs](https://paddlepaddle.github.io/PaddleOCR/) and found no similar bug report.
- [X] I have searched the PaddleOCR [Issues](https://github.com/PaddlePaddle/PaddleOCR/issues) and found no similar bug report.
- [X] I have searched the PaddleOCR [Discussions](https://github.com/PaddlePaddle/PaddleOCR/discussions) and found no similar bug report.

### 🐛 Bug (问题描述)

训练ch_PP-OCRv4几步之后卡住，显存内存一直占用无反应也不继续。
ch_PP-OCRv4_rec.yml:
```Global:
  debug: false
  use_gpu: true
  epoch_num: 200
  log_smooth_window: 20
  print_batch_step: 10
  save_model_dir: ./output/rec_ppocr_v4
  save_epoch_step: 10
  eval_batch_step: [0, 2000]
  cal_metric_during_train: true
  pretrained_model: H:\\paddle_project\\ch_PP-OCRv4_rec_server_train\\best_accuracy.pdparams
  checkpoints:
  save_inference_dir:
  use_visualdl: false
  infer_img: doc/imgs_words/ch/word_1.jpg
  character_dict_path: ppocr/utils/ppocr_keys_v1.txt
  max_text_length: &max_text_length 25
  infer_mode: false
  use_space_char: true
  distributed: true
  save_res_path: ./output/rec/predicts_ppocrv3.txt


Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
    name: Cosine
    learning_rate: 0.001
    values: 0.001
    warmup_epoch: 5
  regularizer:
    name: L2
    factor: 3.0e-05


Architecture:
  model_type: rec
  algorithm: SVTR_LCNet
  Transform:
  Backbone:
    name: PPLCNetV3
    scale: 0.95
  Head:
    name: MultiHead
    head_list:
      - CTCHead:
          Neck:
            name: svtr
            dims: 120
            depth: 2
            hidden_dims: 120
            kernel_size: [1, 3]
            use_guide: True
          Head:
            fc_decay: 0.00001
      - NRTRHead:
          nrtr_dim: 384
          max_text_length: *max_text_length

Loss:
  name: MultiLoss
  loss_config_list:
    - CTCLoss:
    - NRTRLoss:

PostProcess:  
  name: CTCLabelDecode

Metric:
  name: RecMetric
  main_indicator: acc

Train:
  dataset:
    name: MultiScaleDataSet
    ds_width: false
    data_dir: E:\\pic_train\\train/
    ext_op_transform_idx: 1
    label_file_list:
    - E:\\pic_train\\trian.txt
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - RecConAug:
        prob: 0.5
        ext_data_num: 2
        image_shape: [48, 320, 3]
        max_text_length: *max_text_length
    - RecAug:
    - MultiLabelEncode:
        gtc_encode: NRTRLabelEncode
    - KeepKeys:
        keep_keys:
        - image
        - label_ctc
        - label_gtc
        - length
        - valid_ratio
  sampler:
    name: MultiScaleSampler
    scales: [[320, 32], [320, 48], [320, 64]]
    first_bs: 8
    fix_bs: false
    divided_factor: [8, 16] # w, h
    is_training: True
  loader:
    shuffle: true
    batch_size_per_card: 8
    drop_last: true
    num_workers: 1
Eval:
  dataset:
    name: SimpleDataSet
    data_dir: E:\\pic_train\\test/
    label_file_list:
    - E:\\pic_train\\test.txt
    transforms:
    - DecodeImage:
        img_mode: BGR
        channel_first: false
    - MultiLabelEncode:
        gtc_encode: NRTRLabelEncode
    - RecResizeImg:
        image_shape: [3, 48, 320]
    - KeepKeys:
        keep_keys:
        - image
        - label_ctc
        - label_gtc
        - length
        - valid_ratio
  loader:
    shuffle: false
    drop_last: false
    batch_size_per_card: 4
    num_workers: 1
````
log:
```[2024/08/21 14:29:09] ppocr WARNING: The pretrained params backbone.stages.3.blocks.0.aggregation_conv.bn.weight not in model
[2024/08/21 14:29:09] ppocr WARNING: The pretrained params backbone.stages.3.blocks.0.aggregation_conv.bn.bias not in model
[2024/08/21 14:29:09] ppocr WARNING: The pretrained params backbone.stages.3.blocks.0.aggregation_conv.bn._mean not in model
[2024/08/21 14:29:10] ppocr WARNING: The pretrained params backbone.stages.3.blocks.0.aggregation_conv.bn._variance not in model
[2024/08/21 14:29:10] ppocr WARNING: The pretrained params backbone.stages.3.blocks.0.att.conv.weight not in model
[2024/08/21 14:29:10] ppocr WARNING: The pretrained params backbone.stages.3.blocks.0.att.conv.bias not in model
[2024/08/21 14:29:10] ppocr WARNING: The shape of model params head.ctc_encoder.encoder.conv1.conv.weight [60, 480, 1, 3] not matched with loaded params head.ctc_encoder.encoder.conv1.conv.weight [128, 1024, 1, 3] !
[2024/08/21 14:29:10] ppocr WARNING: The shape of model params head.ctc_encoder.encoder.conv1.norm.weight [60] not matched with loaded params head.ctc_encoder.encoder.conv1.norm.weight [128] !
[2024/08/21 14:29:10] ppocr WARNING: The shape of model params head.ctc_encoder.encoder.conv1.norm.bias [60] not matched with loaded params head.ctc_encoder.encoder.conv1.norm.bias [128] !
[2024/08/21 14:29:10] ppocr WARNING: The shape of model params head.ctc_encoder.encoder.conv1.norm._mean [60] not matched with loaded params head.ctc_encoder.encoder.conv1.norm._mean [128] !
[2024/08/21 14:29:10] ppocr WARNING: The shape of model params head.ctc_encoder.encoder.conv1.norm._variance [60] not matched with loaded params head.ctc_encoder.encoder.conv1.norm._variance [128] !
[2024/08/21 14:29:10] ppocr WARNING: The shape of model params head.ctc_encoder.encoder.conv2.conv.weight [120, 60, 1, 1] not matched with loaded params head.ctc_encoder.encoder.conv2.conv.weight [120, 128, 1, 1] !
[2024/08/21 14:29:10] ppocr WARNING: The shape of model params head.ctc_encoder.encoder.conv3.conv.weight [480, 120, 1, 1] not matched with loaded params head.ctc_encoder.encoder.conv3.conv.weight [1024, 120, 1, 1] !
[2024/08/21 14:29:10] ppocr WARNING: The shape of model params head.ctc_encoder.encoder.conv3.norm.weight [480] not matched with loaded params head.ctc_encoder.encoder.conv3.norm.weight [1024] !
[2024/08/21 14:29:10] ppocr WARNING: The shape of model params head.ctc_encoder.encoder.conv3.norm.bias [480] not matched with loaded params head.ctc_encoder.encoder.conv3.norm.bias [1024] !
[2024/08/21 14:29:10] ppocr WARNING: The shape of model params head.ctc_encoder.encoder.conv3.norm._mean [480] not matched with loaded params head.ctc_encoder.encoder.conv3.norm._mean [1024] !
[2024/08/21 14:29:10] ppocr WARNING: The shape of model params head.ctc_encoder.encoder.conv3.norm._variance [480] not matched with loaded params head.ctc_encoder.encoder.conv3.norm._variance [1024] !
[2024/08/21 14:29:10] ppocr WARNING: The shape of model params head.ctc_encoder.encoder.conv4.conv.weight [60, 960, 1, 3] not matched with loaded params head.ctc_encoder.encoder.conv4.conv.weight [128, 2048, 1, 3] !
[2024/08/21 14:29:10] ppocr WARNING: The shape of model params head.ctc_encoder.encoder.conv4.norm.weight [60] not matched with loaded params head.ctc_encoder.encoder.conv4.norm.weight [128] !
[2024/08/21 14:29:10] ppocr WARNING: The shape of model params head.ctc_encoder.encoder.conv4.norm.bias [60] not matched with loaded params head.ctc_encoder.encoder.conv4.norm.bias [128] !
[2024/08/21 14:29:10] ppocr WARNING: The shape of model params head.ctc_encoder.encoder.conv4.norm._mean [60] not matched with loaded params head.ctc_encoder.encoder.conv4.norm._mean [128] !
[2024/08/21 14:29:10] ppocr WARNING: The shape of model params head.ctc_encoder.encoder.conv4.norm._variance [60] not matched with loaded params head.ctc_encoder.encoder.conv4.norm._variance [128] !
[2024/08/21 14:29:10] ppocr WARNING: The shape of model params head.ctc_encoder.encoder.conv1x1.conv.weight [120, 60, 1, 1] not matched with loaded params head.ctc_encoder.encoder.conv1x1.conv.weight [120, 128, 1, 1] !
[2024/08/21 14:29:10] ppocr WARNING: The shape of model params head.before_gtc.1.fc.weight [480, 384] not matched with loaded params head.before_gtc.1.fc.weight [1024, 384] !
[2024/08/21 14:29:10] ppocr INFO: load pretrain successful from H:\\paddle_project\\ch_PP-OCRv4_rec_server_train\\best_accuracy
[2024/08/21 14:29:10] ppocr INFO: During the training process, after the 0th iteration, an evaluation is run every 2000 iterations
W0821 14:29:10.216604 18952 gpu_resources.cc:299] WARNING: device:  . The installed Paddle is compiled with CUDNN 8.4, but CUDNN version in your machine is 8.0, which may cause serious incompatible bug. Please recompile or reinstall Paddle with compatible CUDNN version.
[2024/08/21 14:29:14] ppocr INFO: epoch: [1/200], global_step: 10, lr: 0.000000, acc: 0.000000, norm_edit_dis: 0.000002, CTCLoss: 27.758839, NRTRLoss: 7.578993, loss: 35.337833, avg_reader_cost: 0.00889 s, avg_batch_cost: 0.47548 s, avg_samples: 5.2, ips: 10.93626 samples/s, eta: 94 days, 19:48:59, max_mem_reserved: 1452 MB, max_mem_allocated: 1323 MB
[2024/08/21 14:29:17] ppocr INFO: epoch: [1/200], global_step: 20, lr: 0.000000, acc: 0.000000, norm_edit_dis: 0.000002, CTCLoss: 27.867211, NRTRLoss: 7.578993, loss: 35.551966, avg_reader_cost: 0.00000 s, avg_batch_cost: 0.25403 s, avg_samples: 5.2, ips: 20.46978 samples/s, eta: 72 days, 17:51:00, max_mem_reserved: 1452 MB, max_mem_allocated: 1323 MB
[2024/08/21 14:29:19] ppocr INFO: epoch: [1/200], global_step: 30, lr: 0.000000, acc: 0.000000, norm_edit_dis: 0.000002, CTCLoss: 29.002822, NRTRLoss: 7.609347, loss: 36.717669, avg_reader_cost: 0.00010 s, avg_batch_cost: 0.25648 s, avg_samples: 5.6, ips: 21.83389 samples/s, eta: 65 days, 13:06:05, max_mem_reserved: 1460 MB, max_mem_allocated: 1323 MB
[2024/08/21 14:29:22] ppocr INFO: epoch: [1/200], global_step: 40, lr: 0.000000, acc: 0.000000, norm_edit_dis: 0.000002, CTCLoss: 30.541556, NRTRLoss: 7.656050, loss: 38.187738, avg_reader_cost: 0.00020 s, avg_batch_cost: 0.25427 s, avg_samples: 4.9, ips: 19.27077 samples/s, eta: 61 days, 20:04:52, max_mem_reserved: 1460 MB, max_mem_allocated: 1323 MB
[2024/08/21 14:29:25] ppocr INFO: epoch: [1/200], global_step: 50, lr: 0.000000, acc: 0.000000, norm_edit_dis: 0.000002, CTCLoss: 29.404713, NRTRLoss: 7.541368, loss: 36.793039, avg_reader_cost: 0.00000 s, avg_batch_cost: 0.25825 s, avg_samples: 7.1, ips: 27.49324 samples/s, eta: 59 days, 18:28:23, max_mem_reserved: 1460 MB, max_mem_allocated: 1323 MB
[2024/08/21 14:29:27] ppocr INFO: epoch: [1/200], global_step: 60, lr: 0.000000, acc: 0.000000, norm_edit_dis: 0.000002, CTCLoss: 26.885876, NRTRLoss: 7.429597, loss: 34.339380, avg_reader_cost: 0.00049 s, avg_batch_cost: 0.25704 s, avg_samples: 4.8, ips: 18.67392 samples/s, eta: 58 days, 8:26:30, max_mem_reserved: 1460 MB, max_mem_allocated: 1323 MB
[2024/08/21 14:29:30] ppocr INFO: epoch: [1/200], global_step: 70, lr: 0.000000, acc: 0.000000, norm_edit_dis: 0.000002, CTCLoss: 26.464304, NRTRLoss: 7.564415, loss: 33.974035, avg_reader_cost: 0.00010 s, avg_batch_cost: 0.25528 s, avg_samples: 5.7, ips: 22.32822 samples/s, eta: 57 days, 6:55:46, max_mem_reserved: 1460 MB, max_mem_allocated: 1323 MB
[2024/08/21 14:29:32] ppocr INFO: epoch: [1/200], global_step: 80, lr: 0.000000, acc: 0.000000, norm_edit_dis: 0.000002, CTCLoss: 26.650665, NRTRLoss: 7.564415, loss: 34.328520, avg_reader_cost: 0.00010 s, avg_batch_cost: 0.25683 s, avg_samples: 5.3, ips: 20.63590 samples/s, eta: 56 days, 12:43:24, max_mem_reserved: 1460 MB, max_mem_allocated: 1323 MB
[2024/08/21 14:29:35] ppocr INFO: epoch: [1/200], global_step: 90, lr: 0.000000, acc: 0.000000, norm_edit_dis: 0.000002, CTCLoss: 27.309763, NRTRLoss: 7.462480, loss: 34.624249, avg_reader_cost: 0.00029 s, avg_batch_cost: 0.25742 s, avg_samples: 5.4, ips: 20.97701 samples/s, eta: 55 days, 22:52:38, max_mem_reserved: 1460 MB, max_mem_allocated: 1323 MB
[2024/08/21 14:29:38] ppocr INFO: epoch: [1/200], global_step: 100, lr: 0.000000, acc: 0.000000, norm_edit_dis: 0.000002, CTCLoss: 27.614781, NRTRLoss: 7.419657, loss: 34.974487, avg_reader_cost: 0.00010 s, avg_batch_cost: 0.26224 s, avg_samples: 6.9, ips: 26.31203 samples/s, eta: 55 days, 14:06:13, max_mem_reserved: 1460 MB, max_mem_allocated: 1323 MB
[2024/08/21 14:29:40] ppocr INFO: epoch: [1/200], global_step: 110, lr: 0.000000, acc: 0.000000, norm_edit_dis: 0.000002, CTCLoss: 28.031774, NRTRLoss: 7.398565, loss: 35.563786, avg_reader_cost: 0.00000 s, avg_batch_cost: 0.25220 s, avg_samples: 4.7, ips: 18.63585 samples/s, eta: 55 days, 2:33:31, max_mem_reserved: 1460 MB, max_mem_allocated: 1323 MB
[2024/08/21 14:29:43] ppocr INFO: epoch: [1/200], global_step: 120, lr: 0.000000, acc: 0.000000, norm_edit_dis: 0.000002, CTCLoss: 28.989447, NRTRLoss: 7.383796, loss: 36.335142, avg_reader_cost: 0.00010 s, avg_batch_cost: 0.25503 s, avg_samples: 5.1, ips: 19.99778 samples/s, eta: 54 days, 18:03:53, max_mem_reserved: 1460 MB, max_mem_allocated: 1323 MB
[2024/08/21 14:29:45] ppocr INFO: epoch: [1/200], global_step: 130, lr: 0.000000, acc: 0.000000, norm_edit_dis: 0.000002, CTCLoss: 27.300103, NRTRLoss: 7.372406, loss: 34.939890, avg_reader_cost: 0.00010 s, avg_batch_cost: 0.25279 s, avg_samples: 5.5, ips: 21.75728 samples/s, eta: 54 days, 10:03:11, max_mem_reserved: 1460 MB, max_mem_allocated: 1323 MB
[2024/08/21 14:29:48] ppocr INFO: epoch: [1/200], global_step: 140, lr: 0.000000, acc: 0.000000, norm_edit_dis: 0.000002, CTCLoss: 25.740247, NRTRLoss: 7.372406, loss: 33.215651, avg_reader_cost: 0.00010 s, avg_batch_cost: 0.25724 s, avg_samples: 5.3, ips: 20.60297 samples/s, eta: 54 days, 4:42:33, max_mem_reserved: 1460 MB, max_mem_allocated: 1323 MB

### 🏃‍♂️ Environment (运行环境)

windows
python 3.10.11
paddleocr==2.8.1
gpu 1660s
paddlepaddle-gpu==2.6.1.post117

### 🌰 Minimal Reproducible Example (最小可复现问题的Demo)

python tools/train.py --config=./ch_PP-OCRv4_rec.yml

**Comment by Topdu** [comment]
应该是环境的问题，最好不要使用windows环境训练，推理没问题
